assets/src/components/error-screen/index.js

```diff
@@ -12,34 +13,61 @@ import PropTypes from 'prop-types';
 /**
  * Internal dependencies
  */
+import ClipboardButton from '../clipboard-button';
 import './style.css';
 
 /**
  * Screen that shows when an error has broken the application.
  *
  * @param {Object} props Component props.
  * @param {Object} props.error Error object containing a message string.
- * @param {string} props.finishLink The link to return to the admin.
+ * @param {string} props.finishLinkLabel Label of a link to return to the admin.
+ * @param {string} props.finishLinkUrl Url of a link to return to the admin.
+ * @param {string} props.title Custom message title.
  */
-export function ErrorScreen( { error, finishLink } ) {
+export function ErrorScreen( { error, finishLinkLabel, finishLinkUrl, title } ) {
+	const [ hasCopied, setHasCopied ] = useState( false );
+	const { message, stack } = error;
+
 	return (
 		<div className="error-screen-container">
 			<Panel className="error-screen">
 				<h1>
-					{ __( 'The setup wizard has experienced an error.', 'amp' ) }
+					{ title || __( 'Something went wrong.', 'amp' ) }
 				</h1>
-				<p>
-					{ /* dangerouslySetInnerHTML reason: WordPress sometimes sends back HTML in error messages. */ }
-					<span
-						dangerouslySetInnerHTML={ { __html: error.message || __( 'There was an error loading the setup wizard.', 'amp' ) } }
-					/>
-					{ ' ' }
-					{ finishLink && (
-						<a href={ finishLink }>
-							{ __( 'Return to AMP settings.', 'amp' ) }
+
+				{ /* dangerouslySetInnerHTML reason: WordPress sometimes sends back HTML in error messages. */ }
+				<p dangerouslySetInnerHTML={ {
+					__html: message || __( 'There was an error loading the page.', 'amp' ),
```

Comment:
This would probably be good to have a link to the support forum: https://wordpress.org/support/plugin/amp/

Something like this:

```suggestion
					__html: ( message || __( 'There was an error loading the page.', 'amp' ) ) + ' ' + __( 'Please submit details our <a href="https://wordpress.org/support/plugin/amp/" target="_blank" rel="noreferrer noopener">support forum</a>.', 'amp' ),
```

Naturally perhaps `sprintf()` should be used instead.

Reply:
That's a good idea. In 965e9ec a link to the support forum has been added. It's a separate P tag since I'm not sure if every JS error message ends with a full stop or not. With two paragraph tags, it doesn't really matter that much:

![Screenshot 2021-06-21 at 12 51 54](https://user-images.githubusercontent.com/478735/122750976-77fa9800-d28f-11eb-9242-68c7ed0a3737.png)